config.xml

```diff
@@ -1,7 +1,9 @@
 <?xml version='1.0' encoding='utf-8'?>
```

Comment:
Why do you have changes here? Please do another rebase from master to remove this changes.